content/develop/sidebars.ts

```diff
@@ -0,0 +1,172 @@
+module.exports = {
+    develop: [
+      {
+        label: 'Smart Contracts',
+        type: 'link',
+        href: '/smart-contracts/overview'
+      },
```

Comment:
Could we combine this and the WASP entry?

Reply:
So what if we add a Category
And linke to different section in the SC docs for now?
Something like:
```
Smart Contracts/
├── Documentation
├── Node (WASP)
└── Schema tool or Tools in general 🤔 
```